infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -177,4 +200,62 @@ private ConfigMap newConfigMap(String configMapName, Collection<PluginMeta> plug
   private EnvVar asEnvVar(Pair<String, String> envVar) {
     return new EnvVarBuilder().withName(envVar.first).withValue(envVar.second).build();
   }
+
+  private BrokerConfig createBrokerConfig(
+      RuntimeIdentity runtimeId,
+      Collection<PluginMeta> pluginsMeta,
+      List<EnvVar> envVars,
+      String image,
+      Pod pod) {
+
+    BrokerConfig data = new BrokerConfig();
+    data.configMapName = generateUniqueName(CONFIG_MAP_NAME_SUFFIX);
```

Comment:
As for me, brokerConfig variable name would be simpler here

Reply:
ok